tools/metrics/Gopkg.lock

```diff
@@ -3,11 +3,14 @@
 
 [[projects]]
   branch = "master"
+  digest = "1:79b84ded24b1338dedccd54b7f67ec966407ce4634bd5ccb2560da762619387e"
```

Comment:
I'm not sure quite what to *suggest*, but these SHA->SHA changes (throughout Gopkg.lock) seem quite arbitrary. Can we embed any more info around them, and what they actually reflect? e.g. tags, branches, dates? As potentially the next-person-to-work-on-this, I don't have great visibility of when I should be modifying these (or allowing tooling to modify them) next ... !

Reply:
I think these are mostly caused by the newer version of dep and this project wasn't touched in a long time. As the lock file contains internal details we shouldn't care about most of the changes.

Reply:
(Just to make it clear these changes were fully generated by a `dep ensure` command)